packages/wallet-service/tests/db.test.ts

```diff
@@ -3246,7 +3256,7 @@ describe('getPushDeviceSettingsList', () => {
 
     // register wallets that will not be queried
     const loadWallet = (eachDevice) => createWallet(mysql, eachDevice.walletId, XPUBKEY, AUTH_XPUBKEY, 5);
-    await devicesToLoad.forEach(loadWallet);
```

Comment:
😳

Reply:
Was this test even working?

Reply:
It would load the wallets, but I don't think the await was doing much here.